substratools/opener.py

```diff
@@ -174,13 +174,26 @@ def _assert_output_exists(self, path, key):
             raise exceptions.MissingFileError(f"Output file {path} used to save argument `{key}` does not exists.")
 
 
-def load_from_module(path=None, workspace=None):
-    """Load opener interface from path or from python environment.
-
-    Opener can be defined as an Opener subclass or directly has a module.
+def load_from_module(workspace=None) -> Optional[OpenerWrapper]:
+    """Load opener interface.
+    If a workspace is given, the associated opener will be returned. This means that if no
+    opener_path is defined within the workspace, no opener will be returned
+    If no workspace is given, the opener interface will be directly loaded as a module.
 
     Return an OpenerWrapper instance.
     """
+    if workspace is None:
+        # import from module
+        path = None
+
+    elif workspace.opener_path is None:
+        # no opener within this workspace
+        return None
```

Comment:
is it possible? shouldn't we throw an error in that case?

Reply:
It's simply a task without data (aggregation for example)